static/styles/right_sidebar.css

```diff
@@ -77,7 +77,7 @@ $user_status_emoji_width: 24px;
     }
 
     .user_circle {
-        width: 8px;
+        min-width: 8px;
```

Comment:
Do you have a theory for why just `width` is not working here?

Reply:
@amanagr An initial setting of a flex container is [`flex-shrink: 1`](https://www.w3.org/TR/css-flexbox-1/#flex-shrink-property).

This means that a flex item is allowed to shrink, in order to not overflow the container. So either we can add 
`flex-shrink: 0;` This way we can continue using width in Flex container.

Another option, is to use `min-width`, which is respected when using flex.

Reply:
Using either `min-width` or `flex-shrink: 0` will force us to overflow the container. Since the overflow here is only `0.64` pixel, it is not visually evident. So, I guess, we can merge it for now, I am adding it to my TODO list to write better CSS for this area.